change/@azure-msal-angular-84727060-94f8-459e-9e14-400042bcfcc2.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

```suggestion
  "type": "none",
```

Can be `none` for now. We'll make this a patch when Angular 14 is released as stable.